rust/kcl-lib/src/execution/exec_ast.rs

```diff
@@ -2154,14 +2213,43 @@ impl FunctionSource {
                     }
                 }
 
+                let op = if props.include_in_feature_tree {
+                    let op_labeled_args = args
+                        .kw_args
+                        .labeled
+                        .iter()
+                        .map(|(k, arg)| (k.clone(), OpArg::new(OpKclValue::from(&arg.value), arg.source_range)))
+                        .collect();
+                    Some(Operation::KclStdLibCall {
+                        // TODO
+                        name: String::new(),
```

Comment:
I think this should be...

```suggestion
                        name: props.name.clone(),
```

Reply:
Yeah, I'm just working on this...